docs/docs/development/posix-board.md

```diff
@@ -21,18 +21,18 @@ apt install -y gcc-multilib
 ## Building
 
 To do this, you can build ZMK targeting the
-`native_posix` board.
+`native_posix_64` board.
 
 ```
-west build --pristine --board native_posix
+west build --pristine --board native_posix_64
```

Comment:
I've never built native_posix_64 before, but I'm getting a missing keymap error. It looks like a combination of keymap dirs not including the base boards in cmake as well as no default .keymap file (just overlay).

Reply:
Yeah, we don't have a default keymap for the board.... Not sure we really want to have one. Perhaps our better example here would show an example that includes how to do that.